### PR TITLE
Temporarily disable embedded ppdb data decompression

### PR DESCRIPTION
**Pull request by CoffeeFlux**
Will need to put out a new Ubuntu nightly after this is merged, and then they can re-add us to CI.

### CODE REVIEW

**Comment by lambdageek** [comment]
This addresses #16373. We will work on a proper fix, but this at least unblocks Roslyn CI

File: mono/metadata/debug-mono-ppdb.c [hunk 1]
```diff
@@ -172,9 +172,10 @@ mono_ppdb_load_file (MonoImage *image, const guint8 *raw_contents, int size)
 		return NULL;
 	}
 
+// Temporarily disabled to unblock Roslyn
+#if 0//HAVE_SYS_ZLIB || HOST_WIN32
```
Comment (jaykrell) [inline mono/metadata/debug-mono-ppdb.c]: Isn’t Win32 ok?

Comment (CoffeeFlux) [reply mono/metadata/debug-mono-ppdb.c]: Yeah, should be. Will fix.

**Comment by marek-safar** [comment]
@monojenkins backport 2019-08

**Pull request merged**